src/me/StevenLawson/TotalFreedomMod/Commands/Command_tagnyan.java

```diff
@@ -29,6 +30,19 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
 
         final TFM_PlayerData data = TFM_PlayerData.getPlayerData(sender_p);
         data.setTag(tag.toString());
+        
+        if (!TFM_AdminList.isSuperAdmin(sender))
+        {
+            for (String word : Command_tag.FORBIDDEN_WORDS)
+            {
+                if (args[0].toLowerCase().contains(word))
+                {
+                    playerMsg("That tag contains a forbidden word (" + args[0] + ")");
+                    data.setTag(null);
```

Comment:
@LifeAsStranger If you don't modify the tag until it's been verified as "safe", then it's possible. Everything is possible in java, since it is a [turing complete](https://en.wikipedia.org/wiki/Turing_completeness) programming language.